clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -52,6 +65,127 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Gets a load balancer pool for a given region and load balancer UUID.
+   * @param region
+   * @param loadBalancerId
+   * @return
+   */
+  LbPool getLoadBalancerPool(final Region region, final String loadBalancerId) {
```

Comment:
This makes me thinks that region should be a part of all api calls, not just this one ... I have a similar method in #4  but without region. I think this should go into a separate issue.